.github/workflows/bench.yml

```diff
@@ -66,3 +66,58 @@ jobs:
             pr/build/benchmarks/*
             master/bench-${{ matrix.device }}.*
             master/build/benchmarks/*
+
+  frontier:
+      name: Oak Ridge | Frontier (AMD ROCm)
+      if: github.repository == 'MFlowCode/MFC' && needs.file-changes.outputs.checkall == 'true'
```

Comment:
this almost entirely duplicates the code from phoenix... can you make a CI matrix instead to avoid the extra lines? also there is no need to measure performance on CPU cases on Frontier, that's just going to slow down the number of cases queued. i only have 1 frontier runner because that's the max num. of debug jobs one can submit.